code/modules/antagonists/traitor/contractor/items/contractor_pinpointer.dm

```diff
@@ -39,10 +39,10 @@
 
 /obj/item/pinpointer/crew/contractor/attack_self(mob/living/user)
 	if(owner)
-		if(owner != user)
+		if(owner != user.mind.current)
```

Comment:
Why did you change this code for the contractor pinpointer?

Reply:
The contractor pinpointer also should work after a cloning or brain transplant, as that was in the issue report too, unless I'm missing something and this shouldn't be needed?

Reply:
No, I just didn't see the linked issue for pinpointers. This is fine.